frontend/src/js/controllers/profileCtrl.js

```diff
@@ -127,7 +127,13 @@
             anchor.attr({
                 href: 'data:text/json;charset=utf-8,' + encodeURIComponent(JSON.stringify(vm.jsonResponse)),
                 download: 'token.json'
-            })[0].click();
+            })
+
+            // Create Event
+            var ev = document.createEvent("MouseEvents");
+                ev.initMouseEvent("click", true, false, self, 0, 0, 0, 0, 0, false, false, false, false, 0, null);
+                // Fire event
+                anchor[0].dispatchEvent(ev);
```

Comment:
@gautamjajoo It doesn't work on my local. Can you please verify if it is working for you.

Reply:
@RishabhJain2018 It is working on my end. I have checked it once more. Should I make a screencast?

Reply:
@RishabhJain2018 & @gautamjajoo  The changes LGTM. I've tested this out on all browsers(safari, firefox, opera, chrome on MAC OS), worked well on all of them. We can merge this.